sdk/synapse/synapse-access-control/package.json

```diff
@@ -9,6 +9,7 @@
   "dependencies": {
     "@azure/core-paging": "^1.1.1",
     "@azure/core-http": "^1.2.0",
+    "@azure/core-tracing": "1.0.0-preview.10",
```

Comment:
@joheredi - the other change here is that, rather than using the createSpan export that used to be in core-http, I swap over to using the createSpan that's now being exported from core-tracing.

This helps overall with only having a single place that needs to be updated as we continue to update to newer opentelemetry versions.

Reply:
Great, also this will help decoupling from core-http which will make it easier to move to Core V2

Reply:
Just a note: I have added createSpan back to core-http but that's only for backwards compatibility (as we discussed). Everything in tree currently will call through core-tracing.